showcase/go.mod

```diff
@@ -0,0 +1,16 @@
+module showcase_integration
+
+require (
+	cloud.google.com/go/showcase v0.0.0
+	github.com/golang/protobuf v1.2.0
+	github.com/googleapis/gapic-showcase v0.0.11
+	github.com/googleapis/gax-go v0.0.0-20181219185031-c8a15bac9b9f // indirect
+	github.com/googleapis/gax-go/v2 v2.0.3 // indirect
+	google.golang.org/api v0.1.0
+	google.golang.org/genproto v0.0.0-20190219182410-082222b4a5c5
+	google.golang.org/grpc v1.18.0
+)
+
+replace cloud.google.com/go/showcase => ./gen/cloud.google.com/go/showcase
+
+replace github.com/googleapis/gapic-showcase => ./gen/github.com/googleapis/gapic-showcase
```

Comment:
Why do we need this? Shouldn't we just depend on a specific version of `github.com/googleapis/gapic-showcase`?

Reply:
Oh, it's because we don't actually have that package do we :P

Reply:
Well actually you have a point. This is the base module for `github.com/googleapis/gapic-showcase/server/genproto`. We could depend on the tagged release.

Idk why but i got it in my head that these files should be generated at testing time as well.

Reply:
Oh right that would make sense.

Reply:
Removed it & the stub generation as well. Depending on the tagged version now.